postgres/src/test/java/org/jdbi/v3/postgres/TestInetAddressPg.java

```diff
@@ -13,19 +13,72 @@
  */
 package org.jdbi.v3.postgres;
 
+import java.net.Inet4Address;
+import java.net.InetAddress;
+
 import de.softwareforge.testing.postgres.junit5.EmbeddedPgExtension;
 import de.softwareforge.testing.postgres.junit5.MultiDatabaseBuilder;
+import org.jdbi.v3.core.Handle;
+import org.jdbi.v3.core.Sql;
 import org.jdbi.v3.core.argument.TestInetAddressH2;
 import org.jdbi.v3.core.junit5.PgDatabaseExtension;
+import org.jdbi.v3.testing.junit5.JdbiExtension;
+import org.junit.jupiter.api.Test;
 import org.junit.jupiter.api.extension.RegisterExtension;
 
-public class TestInetAddressPg extends TestInetAddressH2 { { dbExtension = PgDatabaseExtension.instance(pg).withPlugin(new PostgresPlugin()); }
+import static org.assertj.core.api.Assertions.assertThat;
+
+public class TestInetAddressPg extends TestInetAddressH2 {
+
+    {
+        dbExtension = PgDatabaseExtension.instance(pg).withPlugin(new PostgresPlugin());
+    }
 
     @RegisterExtension
     public static EmbeddedPgExtension pg = MultiDatabaseBuilder.instanceWithDefaults().build();
 
+    @RegisterExtension
+    public JdbiExtension pgExtension = JdbiExtension.postgres(pg).withPlugin(new PostgresPlugin());
```

Comment:
this should not be needed. this test extends a test in the core module, so it uses the core specific (non-testing) test classes. dbExtension should work just fine, no need to bring up another Jdbi.